app.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+
+const { getCategories } = require('./controllers/controllers.js');
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/categories', getCategories);
+
+app.all('/*', (req, res, next) => {
+  res.status(404).send({ msg: 'Path not found' });
+});
+
+app.use((err, req, res, next) => {
+  res.status(400).send({msg: err.msg})
```

Comment:
We are not using the 400 error currently, so let's remove this for the time being including the associated catch block :)